src/components/ActivityListItem.js

```diff
@@ -27,3 +26,9 @@ const DivisionLine = styled.div`
   height: 1px;
   margin: 50px 0;
 `
+
+const StyledLi = styled.li`
```

Comment:
Ich hab mich erst gefragt wo die 100px unten hin sind und dann gemerkt, dass sie durch dein Column-Reverse nun unter dem obersten List-Item hängen ;) 
<img width="441" alt="Screen Shot 2020-07-17 at 17 02 15 PM" src="https://user-images.githubusercontent.com/65187999/87801031-72126080-c84f-11ea-88b9-e12d067ff4c1.png">

Falls du unten mehr margin möchtest könntest du das in deiner Main in App.js definieren: 
```
const Main = styled.main`
  overflow-y: scroll;
  &::after {
    content: '';
    display: block;
    height: 40px;
  }

```
40px oder je nachdem wieviel bei dir gut aussieht ;)

Reply:
Da habe ich bei dem reverse nicht dran gedacht. Danke für den Umsetzungshinweis in der App.js!